### ISSUE TITLE
Calendar improvements.

### ISSUE THREAD

**Issue by MilenaCh** [telerik/kendo-themes#254]
- [x] [telerik/kendo#9038] The calendar's content flickers when switch from one view to another https://www.screencast.com/t/MbXNflZIzpk
- [x] [#286] Focus style should be applied when focus some cell via keyboard navigation.
http://kendobuild-windows.openstack.progress.com/kendo-ui/calendar/keyboard-navigation - the user is not able to see which is the currently focused cell.
- [x] [#297] Popup icons changed their orientation on click  https://www.screencast.com/t/yW2Osvuy
- [x] ~~Hover style should be applied only for the icon, which is currently hovered, but no for the both popup icons in DateTimePicker.~~ The current behavior is as expected. However, we should add active styles for these buttons - #352.
- [x] [#434] ~~'today' link beneath the dates should use primary color (purple), not link color (pink): https://github.com/telerik/kendo-ux/blob/master/themes/material/calendar-jquery.psd~~ The color should be pink(check the new designs).
- [x] [#429] calendar navigation buttons have mixed styles, they should behave like flat button (all states should be various opacity of text color), not hovered-gray, selected - purple:
![1](https://user-images.githubusercontent.com/16577622/44983945-c413ab00-af83-11e8-8a2e-d6d7a863bb9a.png)
